ofnetMaster.go

```diff
@@ -174,7 +175,7 @@ func (self *OfnetMaster) EndpointAdd(ep *OfnetEndpoint, ret *bool) error {
 			err := client.Call("OfnetAgent.EndpointAdd", ep, &resp)
 			if err != nil {
 				log.Errorf("Error adding endpoint to %s. Err: %v", node.HostAddr, err)
-				return err
+				// Continue sending the message to other nodes
```

Comment:
Yeah, we don't have any recovery for those.. Since they are bugs :)
But, following route-distribution philosophy, forwarding-table programming failure on one router should not be propagated to other routers..